packages/bruno-app/src/utils/importers/postman-collection.js

```diff
@@ -54,6 +54,23 @@ const convertV21Auth = (array) => {
   }, {});
 };
 
+const constructUrl = (url) => {
+  let urlStr = '';
+  if (typeof url === 'string') {
+    urlStr = url;
+  } else if (url.raw) {
+    urlStr = url.raw;
+  } else {
+    const { protocol, host, path, port, query } = url;
+    const hostStr = Array.isArray(host) ? host.join('.') : host;
+    const pathStr = Array.isArray(path) ? path.join('/') : path;
+    const portStr = port ? `:${port}` : '';
+    const queryStr = query && query.length > 0 ? `?${query.map((q) => `${q.key}=${q.value}`).join('&')}` : '';
+    urlStr = `${protocol}://${hostStr}${portStr}/${pathStr}${queryStr}`;
+  }
+  return urlStr;
+};
```

Comment:
Check this case please, this is losing hash section of url:
```
"item": [
		{
			"name": "Request",
			"request": {
				"method": "GET",
				"header": [],
				"url": {
					"raw": "https://localhost/param1/param2?param3&param4=b&param5=(size=small;color=red)#this-is-framgment",
					"protocol": "https",
					"host": [
						"localhost"
					],
					"path": [
						"param1",
						"param2"
					],
					"query": [
						{
							"key": "param3",
							"value": null
						},
						{
							"key": "param4",
							"value": "b"
						},
						{
							"key": "param5",
							"value": "(size=small;color=red)"
						}
					],
					"hash": "this-is-framgment"
				}
			},
			"response": []
		}
	]
```

Reply:
Thank you for the review. I believe we should not include the hash values in the URL. As per Wikipedia, 

> "The fragment identifier functions differently to the rest of the URI: its processing is exclusively client-sided with no participation from the web server" \
— <cite>[Wikipedia](https://www.wikiwand.com/en/articles/Fragment_identifier)<cite>